main.py

```diff
@@ -33,6 +33,17 @@ def read_env_variables():
     value = os.getenv('TOTP') or None
     TOTP = None if value is None else pyotp.TOTP(value)
 
+    if EMAIL:
+        try:
+            EMAIL = re.search("[^@ \t\r\n]+@[^@ \t\r\n]+\.[^@ \t\r\n]+", EMAIL)[0]
```

Comment:
you can add as many `.com` as you want to your email and it will still be valid with this expression

Reply:
It will also accept all kind of special characters.

Reply:
Perhaps its not even worth adding this kind of validation to the email. What are your thoughts?

Reply:
Odd the "re" was dropped. Still worked as I was testing.
As for if we need it, perhaps you're right.. This is an email validator that works "most of the time" (https://ihateregex.io/expr/email). Up to you really, I just added it to mine

Reply:
Happens 😄
I suggest dropping this change. Here's why. This is a small project and its execution it's meant to be automated, which means, after setting the trigger/cloudwatch event once (the email is set in this spet) you will not look at it again. For that reason I don't see a point to check the email on every execution.

Reply:
Yea that's fine.. live and learn